course-mw/course-actors-common/src/main/java/org/sunbird/models/course/batch/CourseBatch.java

```diff
@@ -213,4 +215,21 @@ public String getOldUpdatedDate() {
   public void setOldUpdatedDate(String updatedDate) {
     this.oldUpdatedDate = updatedDate;
   }
+
+  public Map<String, Object> getBatchAttributes() {
+    return batchAttributes;
+  }
+
+  public void setBatchAttributes(Map<String, Object> batchAttributes) {
+    this.batchAttributes = batchAttributes;
+  }
+
+  public Integer getCurrentBatchSize() {
+    return currentBatchSize;
+  }
+
+  public void setCurrentBatchSize(Integer currentBatchSize) {
+    this.currentBatchSize = currentBatchSize;
+  }
```

Comment:
this is poor design... you have batchAttriubtes then why you need currentBatchSize as additional attribute in the object??
the request body you can add the batchSize inside the batchAttributes -- so that it is easy in future to add more parameters...